pkg/ebpf/events_enrich.go

```diff
@@ -89,7 +89,7 @@ func (t *Tracee) enrichContainerEvents(ctx gocontext.Context, in <-chan *trace.E
 				cgroupId := uint64(event.CgroupID)
 				// CgroupMkdir: pick EventID from the event itself
 				if eventID == events.CgroupMkdir {
-					cgroupId, _ = parse.ArgVal[uint64](event, "cgroup_id")
+					cgroupId, _ = parse.ArgVal[uint64](event.Args, "cgroup_id")
 				}
 				// CgroupRmdir: clean up remaining events and maps
 				if eventID == events.CgroupRmdir {
```

Comment:
We could add an enrich call at the end of the control plane logic so the code here could take less time (since enrichment would already be done), that's a good idea. 
But the change done here is just to align with the `parse.ArgVal` function signature change.

Reply:
By starting the `EnrichCgroupInfo` sooner (in the control plane), we will help the pipeline (that might receive same related cgroupid events a bit later because of its channel buffer). I think its a good idea.

Reply:
Added now, will test to see if it works right, please review it as well @rafaeldtinoco.

Reply:
So if we now enrich using the control plane messages - do we still need to perform the code below (currently the lines below are not changed in this PR)